### ISSUE TITLE
Only show activation banner for user who activated the plugin

### ISSUE THREAD

**Issue by DevinWalker** [WordImpress/Give#1036]
I would like to do some additional improvements like better handling of the banner when activating multiple add-ons and only display for the user activating the banner, not a newly created user.

- When multiple add-ons are activated, we should unify the activation messages so they don't stack annoyingly. UI needed for this...
- The banner should only display for the user activating the plugin, right now if you create a new user or log in to an existing user and view the plugins screen the banner displays improperly which confuses the user since they never activated it.

**Comment by ramiy** [comment]
@Desertsnowman I've improved the add-ons activation code. Some add-ons had activation banners, others did not. Those who did, they all had different code.

Some of the add-ons used their own activation banners, now they all use the main **Give_Addon_Activation_Banner()** class from Give core. Each add-on just need to set it's own `$args` and the `admin_notices` action fallback function.

I already updated 10 add-ons, planning to finish all of them.

![activation-banners](https://cloud.githubusercontent.com/assets/576623/18731039/29f779a4-8062-11e6-980b-5b0c22ede410.png)

Not sure how you plan to unify different messages into one. Each one has it's own user-meta field to update, for dismissing the message.
